core/src/main/java/me/leoko/advancedban/utils/Punishment.java

```diff
@@ -110,7 +112,6 @@ public void create(boolean silent) {
                     for (String str : getLayout()) {
                         mi.sendMessage(p, str);
                     }
-                PunishmentManager.get().getLoadedPunishments(false).add(this);
```

Comment:
Revert this change, the correct placement is in: https://github.com/DevLeoko/AdvancedBan/blob/2af7d939f608b22715d65a45edb678c0849e5dbc/core/src/main/java/me/leoko/advancedban/utils/Punishment.java#L113